### PR TITLE
[W5.4][T16-A4]Lin Jiachao

### PR DESCRIPTION
**Pull request by lincredibleJC**
Added a new function to search for persons by keywords in their tags.
Added Unit tests for the FindTagsCommand class
Updated User Guide

### CODE REVIEW

**Comment by thegreatkang** [comment]
Good, maybe can try to combine with findCommand such that any keywords will check and compare with both name AND tags.

File: src/seedu/addressbook/commands/FindTagCommand.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.Tag;
+
+import java.util.*;
```
Comment (Skaty) [inline src/seedu/addressbook/commands/FindTagCommand.java]: According to our Java style guide: "Imported classes should always be listed explicitly."

File: test/java/seedu/addressbook/parser/ParserTest.java [hunk 2]
```diff
@@ -12,17 +12,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (Skaty) [inline test/java/seedu/addressbook/parser/ParserTest.java]: Same here.

File: src/seedu/addressbook/commands/FindTagCommand.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+public class FindTagCommand extends Command {
```
Comment (Skaty) [inline src/seedu/addressbook/commands/FindTagCommand.java]: Add header comments for this class, as it is non-trivial code.

**Pull request closed without merging**